lib/AST/Type.cpp

```diff
@@ -812,6 +812,28 @@ bool TypeBase::isStdlibType() {
   return false;
 }
 
+bool TypeBase::isCGFloatType() {
+  auto *NTD = getAnyNominal();
+  if (!NTD)
+    return false;
+
+  auto *DC = NTD->getDeclContext();
+  if (!DC->isModuleScopeContext())
+    return false;
+
+  auto *module = DC->getParentModule();
+  // on macOS `CGFloat` is part of a `CoreGraphics` module,
```

Comment:
```suggestion
  // On macOS `CGFloat` is part of a `CoreGraphics` module,
```

Reply:
@xwu I'm just going to lift this comment to the declaration as well to make it clear where `CGFloat` could be.

Reply:
✅